ports/glutin/window.rs

```diff
@@ -605,15 +607,21 @@ impl WindowMethods for Window {
     }
 
     fn set_page_title(&self, title: Option<String>) {
+        let fallback_title = match self.current_url.borrow_mut().clone() {
```

Comment:
`if let Some (ref current_url) = self.current_url.borrow().as_ref()` 
and
`if let &Some (ref current_url) = self.current_url.borrow()`
should both work.